src/kind.ts

```diff
@@ -0,0 +1,74 @@
+import { Job } from "@brigadecore/brigadier";
+
+export const kindJobImage = "radumatei/golang-kind:1.11-0.4";
```

Comment:
Does this image have a backing GH repo?  We should add a link to the repo somewhere, in case a fix/feature is motivated from code changes in this repo...

Reply:
We should collect all of the images we use in a repo here or in `deislabs`.